iOS_SDK/OneSignalSDK/Source/OSPermission.m

```diff
@@ -81,7 +81,7 @@ - (instancetype)copyWithZone:(NSZone*)zone {
 
 - (void)setHasPrompted:(BOOL)inHasPrompted {
     if (_hasPrompted != inHasPrompted)
-        [OneSignalUserDefaults.initStandard saveBoolForKey:HAS_PROMPTED_FOR_NOTIFICATIONS withValue:true];
+        [OneSignalUserDefaults.initStandard saveBoolForKey:OSUD_WAS_PROMPTED_FOR_NOTIFICATIONS withValue:true];
```

Comment:
It seems `OSUD_WAS_PROMPTED_FOR_NOTIFICATIONS` is save here but don't see it read anywhere...

Reply:
Looking into this now, might need to go back in time for this one.

Reply:
Deleting this `saveBoolForKey`, but keeping the key for possible future usage